### PR TITLE
Edit user guide to follow the format and change it based on the feedback.

### LINKED ISSUE #144: Add preface to the quick start section of UG
Provide a quick start demo with pictures

### PR DESCRIPTION
**Pull request by GilbertTan19**

### CODE REVIEW

**Review by ChenXJ98 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -24,30 +25,32 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
    
-   * **`list`** : Lists all FlashCards.
-   
-   * **`open`**`1` : Opens the 1st question shown in the current list.
+   * **`add`**`q/Plants give out ___ when they photosynthesise? ans/Oxygen t/Biology` :  Adds an open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`.
```
Comment (joshtyf) [inline docs/UserGuide.md:28]: Could you change it from

> Adds an open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`.

to "Adds a flashcard with an open ended question `Plants give out ___ when they photosynthesise?`. The flashcard will have an answer  `Oxygen` and has a tag `Biology`."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -24,30 +25,32 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
    
-   * **`list`** : Lists all FlashCards.
-   
-   * **`open`**`1` : Opens the 1st question shown in the current list.
+   * **`add`**`q/Plants give out ___ when they photosynthesise? ans/Oxygen t/Biology` :  Adds an open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`. 
+      
+   * **`addmcq`**`q/Plants give out ___ when they photosynthesise? ans/1 c/Oxygen c/Carbon c/Carbon dioxide` :  Adds a multiple choice question `Plants give out ___ when they photosynthesise?` with 3 options `Oxygen`, `Carbon`, `Carbon dioxide` and with answer `Oxygen`.
```
Comment (joshtyf) [inline docs/UserGuide.md:30]: Could you change it from 

> Adds a multiple choice question `Plants give out ___ when they photosynthesise?` with 3 options `Oxygen`, `Carbon`, `Carbon dioxide` and with answer `Oxygen`.

to "Adds a flashcard with a multiple choice question `Plants give out ___ when they photosynthesise?` with 3 options `Oxygen`, `Carbon`, `Carbon dioxide`. The flashcard will have an answer `Oxygen`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -24,30 +25,32 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
...
-   * **`add`**`q/Plants give out ___ when they photosynthesise? ans/Oxygen t/Biology` :  Adds a open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`. 
+   * **`open`**`1` : Opens the 1st question shown in the current list.
    
-   * **`addmcq`**`q/Plants give out ___ when they photosynthesise? ans/1 c/Oxygen c/Carbon c/Carbon dioxide` :  Adds a multiple choice question `Plants give out ___ when they photosynthesise?` with 3 options `Oxygen`, `Carbon`, `Carbon dioxide` and with answer `Oxygen`.
+   * **`edit`**`1 ans/Edited answer` : Edit the answer of the first flashcard on the list to become `Edited answer`.
    
-   * **`test`**`1 ans/Example answer` : Tests the 1st question shown in the current list with `Example answer` as the answer.
+   * **`list`** : Lists all FlashCards.
    
    * **`find`** `find t/MCQ q/What CS2103T q/is t/GoodQuestion` : Finds all Flashcards tagged to the tag `MCQ` and `GoodQuestion` and has keywords `What`, `CS2103T` and `is` in question.
```
Comment (joshtyf) [inline docs/UserGuide.md:38]: is there a missing `q/` before the keyword `CS2103T`?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -72,9 +75,40 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 
 Shows a message explaning how to access the help page.
 
+Format: `help`
+
 ![help message](images/helpMessage.png)
 
-Format: `help`
+### Creating a new flashcard : You can add a new flashcard to the list.
```
Comment (joshtyf) [inline docs/UserGuide.md:82]: The preface of the feature should not in the header. Refer to my implementation of the delete section

File: docs/UserGuide.md [hunk 5]
```diff
@@ -72,9 +75,40 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 
 Shows a message explaning how to access the help page.
 
+Format: `help`
```
Comment (joshtyf) [inline docs/UserGuide.md:78]: The format should be moved to the header

**Review by joshtyf — COMMENTED** [review]
Sorry Gilbert :( You have to make more changes before I can approve this :(

**Pull request merged**